hoot-services/src/main/java/hoot/services/controllers/ingest/ImportResource.java

```diff
@@ -246,10 +246,9 @@ public Response processFileUpload(@Context HttpServletRequest request,
                     etlName, noneTranslation, debugLevel, finalUploadClassification, this.getClass(), user);
             workflow.add(importCommand);
 
-            if(folderId != null) {
-                InternalCommand setFolderCommand = updateParentCommandFactory.build(jobId, Long.parseLong(folderId), etlName, user, this.getClass());
-                workflow.add(setFolderCommand);
-            }
+            long parentFolderId = folderId != null ? Long.parseLong(folderId) : 0;
```

Comment:
This change is making it use the root folder if not specified vs not setting a folder id on the dataset at all?

Reply:
Yea and because it is done here, we get to skip all that logic i removed in the FolderResource file that was basically just checking if it exists in the folderMapMappings table and adding there if it didnt